rpc-component/lib/schemata.py

```diff
@@ -0,0 +1,154 @@
+from functools import partial
+from operator import itemgetter
```

Comment:
This not used anywhere?

Reply:
There were a couple of unused imports, I've cleaned that up now.